contributions/api/controllers/contributions.py

```diff
@@ -45,6 +45,11 @@
 coll_reviewer = db_contribution[cfg.REVIEWER_COLL_NAME]
 notification_enabled = cfg.NOTIFICATION_ENABLED
 
+if notification_enabled:
```

Comment:
Would it be better to move these lines of code to the contributions_rest_service.py file?

Reply:
The established connection (object) will need to be saved and used in contributions.py. I'm not quite sure on how to do that now. 
It will be nice if we have establish connection and quit connection calls from contributions_rest_service.py. Just not sure on the implementation.